config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-e2e.yaml

```diff
@@ -0,0 +1,42 @@
+periodics:
```

Comment:
I just noticed that we are defining periods both here and in config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

Reply:
Hi @detiber,

Oh, can there only be one TLD per sub-dir? I haven't worked with Prow's broken-apart config structure before -- only the giant config file.

Reply:
I'm not sure if that is a limitation, but it might make discovery for developers more difficult to spread out periodic job definitions among multiple files.

Reply:
Hi @detiber,

Hmm, I disagree. The entire point of this structure is to allow the spreading out of job definitions among multiple files. If you're firm on this then I'd just as soon get rid of the existing definition as it just verifies it can check out the account from Boskos and nothing more.

Reply:
Not firm at all, lets roll with it and we can re-assess discoverability at a later date.

Reply:
Hi @detiber,

Sounds good. I agree discoverability is important. As there are only three files with one that could likely go away, I feel like this works for now. The choice becomes does one pivot on the file name for the type of tests it includes regardless of the test signal (periodic, pre, post) or does a file contain multiple test signals but all belonging to a single test *type*, such as e2e, non-e2e, etc.

I looked in the other directories for examples, and while files do seem to break down between pre/post, I didn't see files specifically for periodics. That said, I also did not do an exhaustive search.

Reply:
> Oh, can there only be one TLD per sub-dir? I haven't worked with Prow's broken-apart config structure before -- only the giant config file.

You can have multiple yaml files per sub dir, and you can have nested subdir as well, we walks the entire tree :-)